src/Components/v2/ArtworkFilter/ArtworkFilters/SizeRangeFilters.tsx

```diff
@@ -0,0 +1,52 @@
+import { LabeledRange } from "@artsy/palette"
+import React, { FC } from "react"
+import { useFilterContext } from "../ArtworkFilterContext"
+
+import {
+  MAX_HEIGHT,
+  MAX_WIDTH,
+  MIN_HEIGHT,
+  MIN_WIDTH,
+} from "../Utils/rangeToTuple"
+
+export const SizeRangeFilters: FC = () => {
+  const filterContext = useFilterContext()
+
+  const [initialMinHeight, initialMaxHeight] = filterContext.rangeToTuple(
```

Comment:
I think we we can defer to the `rangeToTuple` util vs moving through the context